variables.md

```diff
@@ -0,0 +1,204 @@
+# VARIABLES
+
+A variable is a container for a data which provides a way of labeling data with a descriptive name. Variables label and store data in memory,update information stored in it and Reference or “get” information stored in it.
+
+It is important to distinguish that variables are not values; they contain values and represent them with a name.
+
+
+## Creating a Variable
+
+### ```var```
+
+ Before the introduction of ES6 version of javascript, Programmers use the ```var``` keyword to create a variable.
+
+ ```
+ var myName = 'Arya';
+ console.log(myName);// Output: Arya
+ ```
+
+### ```let```
+
+ The ```let``` keyword was introduced in ES6. The ```let``` keyword signals that the variable can be reassigned a different value.
+
+ ```
+ let meal = 'Enchiladas';
+ console.log(meal); // Output: Enchiladas
+ meal = 'Burrito';
+ console.log(meal); // Output: Burrito
+ ```
+
+#### Similarities of ```var``` and ```let``` keywords
+- Both can be used to declare a variable without assigning the variable a value. In such a case, the variable will be automatically initialized with a value of ```undefined```:
+ ```
+ let price;
+ console.log(price); // Output: undefined
+ price = 350;
+ console.log(price); // Output: 350
+ ```
+
+ ```
+ var price;
+ console.log(price); // Output: undefined
+ price = 350;
+ console.log(price); // Output: 350
+ ```
+ If we don’t assign a value to a variable declared using the ```let``` keyword, it automatically has a value of undefined.
+
+
+- From the above similarity, it can be established that the value of both variables can be reassigned.
+
+
+#### Disimilarities between ```var``` and ```let``` keywords
+
+the difference between them is that ```var``` is function scoped and ```let``` is block scoped. example:
+
+```
+console.log(x);
+var x=5;
+console.log(x); //Output: undefined, 5
+```
+
+if an undeclared variable is logged to the console and later declared with ```var``` and assigned a value, it will return undefine and the value assigned to it.
+
+an error will be returned if let is used in declaring such variable. example:
+
+
+```
+console.log(x); // output: ReferenceError: x is not defined
+let x=5;
+console.log(x); //Output: 5
+```
+
```

Comment:
Are you sure this is an explanation of `var` been a function scope and `let` been a block scoped?
Can you post your source?
You didn't use any function or block in this example, so how did you illustrate the difference?

I would recommend reading on the difference of `var` and `let`